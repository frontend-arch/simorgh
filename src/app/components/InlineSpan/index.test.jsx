import React from 'react';
import { latin, arabic } from '@bbc/gel-foundations/scripts';
import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
import Inline from './index';

describe('Inline', () => {
  shouldMatchSnapshot(
    'english inline span in persian service article',
    <Inline lang="en" script={arabic} service="persian">
      This is text in a Inline.
    </Inline>,
  );

  shouldMatchSnapshot(
    'persian inline span in english service article',
    <Inline lang="fa" script={latin} service="news">
      بعض محتوى النص
    </Inline>,
  );
});
