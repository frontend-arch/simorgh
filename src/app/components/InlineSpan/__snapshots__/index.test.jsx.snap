// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inline english inline span in persian service article 1`] = `
.c0 {
  padding-bottom: 1.5rem;
  margin: 0;
}

<span
  class="c0"
  lang="en"
>
  This is text in a Inline.
</span>
`;

exports[`Inline persian inline span in english service article 1`] = `
.c0 {
  padding-bottom: 1.5rem;
  margin: 0;
}

<span
  class="c0"
  lang="fa"
>
  بعض محتوى النص
</span>
`;
